extra/1-radio-stations.js

```diff
@@ -26,42 +36,58 @@
  */
 // `getStations` goes here
 
+function getStations() {
+
+  
+  let station = [];
+
+	let frequency= getAllFrequencies();
+	for (let i = 0; i < frequency.length; i++) {
+		if (isRadioStation(frequency[i])) {
```

Comment:
Maybe assign `frequency[i]` to a variable called something like currentFrequency, to make the code more understandable.